### PR TITLE
Consistent default and build steps with other projects

### PR DESCRIPTION
**Pull request by georgecrawford**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "description": "A WebdriverIO plugin. Helper module to run WebdriverIO commands synchronously.",
   "main": "build/index.js",
   "scripts": {
-    "test": "grunt"
+    "test": "./node_modules/.bin/grunt"
```
Comment (christian-bromann) [inline package.json]: as far as I know this is not necessary in the package.json .. npm automatically uses the binary in node_modules

**Comment by christian-bromann** [comment]
One comment otherwise LGTM

File: package.json (hunk 1 above)
Comment (georgecrawford) [inline package.json]: OK, then we should change the other packages, or just ignore :)

**Comment by georgecrawford** [comment]
Updated

**Pull request merged**